### ISSUE TITLE
"Quote selected text" breaks smilies under unclear circumstances

### ISSUE THREAD

**Issue by m4z** [SimpleMachines/SMF#5848]
#### Description
In some of my forums, when selecting text containing smilies, then clicking "Quote selected text", and then previewing, instead of the image, the full URL is displayed (in parenthesis):

Working forum:
![smf-quote-selected-smiley-working](https://user-images.githubusercontent.com/387675/67634342-d2750280-f8ba-11e9-896a-9fa4e9ce4e1b.png)

Broken forum:
![smf-quote-selected-smiley-broken](https://user-images.githubusercontent.com/387675/67634344-d6a12000-f8ba-11e9-9b53-5671955e85fd.png)

The issue here is **not** that the first quote doesn't contain text; I've tested that too, and only brainfarted when doing the screenshots.
I haven't yet figured out what is causing this different behavior, but I've excluded the following as a cause:
* Mods
* Enabling custom smilies
* Actually _having_ custom smilies

### Steps to reproduce
0. ???
1. Select some text to quote, containing a smiley
2. Click "Quote selected text"
3. Hit Preview
4. Smiley isn't rendered

### Environment (complete as necessary)
- Version/Git revision: HEAD of ~2019-10-10
- Database Type: Postgres
- Database Version: 9.x
- PHP Version: 5.6.x

### Additional information/references

**Comment by MissAllSunday** [comment]
This largely depends on the browser you are using so, what browser are you using? if possible, the exact version.

**Comment by m4z** [comment]
I noticed this with the latest Google Chrome (will provide details later) on W10 Pro N 64bit.

**Comment by m4z** [comment]
Also applies to (all on LinuxMint 19.1 (64-bit)):
* Firefox Developer Edition "70.0b13 (64-bit)"
* Chromium "Version 77.0.3865.90 (Official Build) Built on Ubuntu , running on LinuxMint 19.1 (64-bit)"
* Google Chrome "Version 78.0.3904.70 (Official Build) (64-bit)"

**Comment by m4z** [comment]
Oh my dog… I'm stupid and could've figured this out sooner: In the affected forums (which were the "more productive" ones of my forums), I had disabled (amongst others), the `[img]` BBCode for its GDPR implications[0] (and because there was no need for it by my users).
I brainfarted and didn't notice that "Quote selected text" creates that tag instead of "bypassing" BBC like the original post.

[0]: I don't have an image proxy (yet, I guess?).
